Data/Sys/GameSettings/GMPE01.ini

```diff
@@ -409,4 +428,4 @@ $Minigame: Take a Breather - Mash Only L [gamemasterplc]
 04420818 38a00000
 044207f0 38600000
 e2000001 80008000
-*Mash just L in the minigame Take a Breather.
+*Mash just L in the minigame Take a Breather.
```

Comment:
I don't know if this matters, but github complains that there's no newline at end of file.